source/common/config/well_known_names.h

```diff
@@ -324,5 +324,16 @@ class TransportSocketNameValues {
 
 typedef ConstSingleton<TransportSocketNameValues> TransportSocketNames;
 
+/**
```

Comment:
Please see how I broke out well_known_names in https://github.com/envoyproxy/envoy/pull/3075. Can you replicate that structure when you merge master? Thank you!

Reply:
@mattklein123 OK. I have it on following commits. I hope it is fine to [require ext well-known names for a while](https://github.com/dio/envoy/blob/fc104517bc1e9d164d70490efd860f54e7d0553c/source/common/upstream/BUILD#L120) from upstream health checker (until `redis_health_check` is removed).